### ISSUE TITLE
cs command is not working

### ISSUE THREAD

**Issue by andreyohiienko** [VSCodeVim/Vim#6815]
**Describe the bug**
From version 1.21.0 cs command stopped to work

**To Reproduce**
Instal 1.21.0 version or higher
Go to line or words covered by any of this symbol '"[{(
For example try to change single quotes to double quotes
cs'" in this line
import React from 'react'

**Expected behavior**
cs'" should change 
import React from 'react'
to
import React from "react"
like it was in 1.20.3 versions and below

**Screenshots**
Was in 1.20.3 and below
https://www.screencast.com/t/WNPS3Gut
now 
https://www.screencast.com/t/zwwV7jHS5

Version: 1.57.1 (user setup)
Electron: 12.0.7
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19041
vscodevim: 1.21.3

**Comment by J-Fields** [comment]
Works for me. Can you please provide your settings?

**Comment by sql-koala** [comment]
Yeah, I suspect it has to do with a custom mapping from or to cs.

**Comment by andreyohiienko** [comment]
sure
`{
    "[javascript]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode",
        "editor.formatOnSave": true,
        "editor.tabSize": 2
    },
    "[javascriptreact]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode",
        "editor.formatOnSave": true,
        "editor.tabSize": 8
    },
    "files.trimTrailingWhitespace": true,
    "[markdown]": {
        "files.trimTrailingWhitespace": false
    },
    "files.autoSave": "onFocusChange",
    "vim.easymotion": true,
    "vim.insertModeKeyBindings": [
        {
        "before": [
            "d",
            "f"
        ],
        "after": [
            "<Esc>"
        ]
        }
    ],
    "vim.normalModeKeyBindingsNonRecursive": [
        {
            "before": ["ctrl+l"],
            "after": ["leader", "leader", "leader", "b", "d", "w"]
        },
        {
            "before": [" "],
            "after": ["leader", "leader", "s"]
        },
    ],
    "vim.easymotionMarkerBackgroundColor": "rgba(247, 255, 0, 0.075);",
    "vim.useCtrlKeys": false,
    "vim.handleKeys": {
        "<C-d>": false,
    },
    "vim.easymotionKeys": "hklyuiopnm,qwertzxcvbasdgjf;1234567890",
    "vim.timeout": 125,
    "vim.highlightedyank.enable": true,
    "vim.mouseSelectionGoesIntoVisualMode": false,
    "vim.easymotionDimBackground": true,
    "vim.debug.silent": true,
    "vim.visualstar": true,
    "git.suggestSmartCommit": false,
    "editor.snippetSuggestions": "top",
    "emmet.showSuggestionsAsSnippets": true,
    "emmet.includeLanguages": {
        "blade": "html",
        "blade.php": "html",
        "javascript": "javascriptreact",
        "vue": "html",
        "vue-html": "html",
        "php": "html"
    },
    "emmet.showExpandedAbbreviation": "always",
    "emmet.syntaxProfiles": {
        "javascript": "jsx",
        "blade": "html",
        "blade.php": "html",
        "php": "html",
    },
    "javascript.suggest.autoImports": true,
    "typescript.suggest.autoImports": true,
    "javascript.updateImportsOnFileMove.enabled": "always",
    "typescript.updateImportsOnFileMove.enabled": "always",
    "prettier.trailingComma": "all",
    "task.slowProviderWarning": [
        "npm"
    ],
    "gitlens.gitCommands.skipConfirmations": [
        "fetch:command",
        "stash-push:command",
        "switch:command",
        "switch:menu"
    ],
    "gitlens.gitCommands.closeOnFocusOut": true,
    "gitlens.keymap": "none",
    "workbench.iconTheme": "material-icon-theme",
    "workbench.startupEditor": "newUntitledFile",
    "explorer.confirmDelete": false,
    "vs-kubernetes": {
        "vs-kubernetes.minikube-path.windows": "C:\\Users\\Andrew\\.vs-kubernetes\\tools\\minikube\\windows-amd64\\minikube.exe",
        "vs-kubernetes.helm-path.windows": "C:\\Users\\Andrew\\.vs-kubernetes\\tools\\helm\\windows-amd64\\helm.exe",
        "vs-kubernetes.draft-path.windows": "C:\\Users\\Andrew\\.vs-kubernetes\\tools\\draft\\windows-amd64\\draft.exe"
    },
    "npm.enableRunFromFolder": true,
    "git.confirmSync": false,
    "terminal.integrated.profiles.windows": {
        "PowerShell": {
            "source": "PowerShell",
            "overrideName": true,
            "icon": "terminal-powershell",
            "env": {
                "TEST_VAR": "value"
            }
        }
        },
        "terminal.integrated.defaultProfile.windows": "PowerShell",
    "remote.extensionKind": {
        "ms-azuretools.vscode-docker": "workspace"
    },
    "gitlens.defaultDateFormat": null,
    "gitlens.currentLine.enabled": false,
    "gitlens.hovers.currentLine.over": "line",
    "gitlens.hovers.currentLine.enabled": false,
    "intelephense.environment.includePaths": [
        "C:\\php-integrations\\wordpress",
        "C:\\php-integrations\\advanced-custom-fields"
    ],
    "bladeFormatter.format.indentSize": 2,
    "blade.format.enable": true,
    "diffEditor.ignoreTrimWhitespace": false,
    "merge-conflict.autoNavigateNextConflict.enabled": true,
    "terminal.integrated.tabs.enabled": true,
    "editor.tabSize": 8,
    "editor.cursorStyle": "line",
    "editor.insertSpaces": false,
    "editor.lineNumbers": "on",
    "editor.wordSeparators": "/\\()\"':,.;<>~!@#$%^&*|+=[]{}`?-",
    "editor.wordWrap": "off"
}`

**Comment by sql-koala** [comment]
Hmm, I dont see anything that could explain it, sorry :|

**Comment by andreyohiienko** [comment]
I made Vim extension enabled only and still see this issue. Tell me please what more info needed?

**Comment by andreyohiienko** [comment]
I noticed that it works like with unenabled "Vim: Surround" feature. The same behavior.
@J-Fields
@sql-koala
